src/screens/HomePage/HomePage.tsx

```diff
@@ -165,9 +157,6 @@ export default function HomePage() {
                 showNationalSummary={true}
               />
             </Section>
-            <SectionWrapper>
-              <CriteriaExplanation isMobile={isMobile} />
```

Comment:
i assume we can also delete `CriteriaExplanation.tsx` and `CriteriaExplanation.style.tsx`